docs/reporting-services/change-log-sql-server-reporting-services.md

```diff
@@ -19,6 +19,19 @@ This article describes changes in [!INCLUDE[ssRSnoversion](../includes/ssrsnover
 
 ## SQL Server 2017 Reporting Services 
 
+- *Version 14.0.600.892, Released: August 31, 2018* 
+    - Bug Fixes:
+        - Textbox inside Rectangle causes rectangle not to grow vertically when rc:Toolbar=False and it has long text
+        - Text size is not scaling if pageHeight is less than 0.5 in
+        - Deadlock in SSRS catalog database when used with CRM
+        - Vertically aligned column headers incorrectly displayed when scrolling down in report
+        - Users added to SCOM Reporting Role will have access blocked to SSRS web portal
+        - Thai character is not getting exported correctly in PDF
+        - Browser Role Behavior Change
```

Comment:
I don't know if it's valuable to add such a generic bug fix. I'd leave this one off along w/ the "Updated Mobile Report Runtime" entry at the bottom